### ISSUE TITLE
FAQ Content

### ISSUE THREAD

**Issue by lfantano** [deptofdefense/move.mil#72]
Needs content update for answers to FAQ questions.

**Comment by lfantano** [comment]
http://staging.3iweb3tb2j.us-east-1.elasticbeanstalk.com/faqs

**Comment by jde1984** [comment]
[FAQ.docx](https://github.com/deptofdefense/move.mil/files/1246235/FAQ.docx)

The only thing that needs to be added in here when it's posted to the site are the links to the PPPO locator, move.mil, weight allowance, weight scales, tips, and MCO's.   If we get more FAQ's just let me know and I can dig up the answers quickly.

**Comment by jgarber623-gov** [comment]
Looking through @jde1984's attached `FAQ.docx`, FAQs are organized into sections:

```
Before You Move
D-Day… the Move Is Here!
Tips As you Travel
Destination Delivery
After The Move… Vouchers, Claims and Surveys!
```

On [the FAQs page on beta.move.mil](http://beta.move.mil/faqs), questions aren't organized into sections, but instead use a rudimentary tagging system with a filter UI:

<img width="1316" alt="screen shot 2017-08-28 at 3 46 53 pm" src="https://user-images.githubusercontent.com/27780860/29790504-20306c2a-8c08-11e7-8143-09351d0cc928.png">

The content doc's organization implies a different layout for this page. @lfantano What do you have in mind for the FAQs page based on the new content the existing visual design/layout?

**Comment by lfantano** [comment]
We need to tweak the content @jde1984 provided to match the template. Stay
Tuned!!!

**Comment by jde1984** [comment]
@jgarber623-gov @lfantano - let me know if I can help, I assembled the FAQ's in the same pattern as I did the moving tips... basically from start to finish to help take small bites out of the elephant but feel free to adjust fire.

**Comment by lfantano** [comment]
Ok- I'll take a stab at making a Google Doc with questions and maybe you
can help me get answers?

**Comment by jde1984** [comment]
Sounds good - most of the questions that were in the original beta site were answered in my other FAQ questions, but if I missed anything I can provide the content if you and Jason want to figure out the best way to organize it.

**Comment by Elliottdotgov** [comment]
@jde1984 @lfantano bump - where do we stand on this?

**Comment by jgarber623-gov** [comment]
Closed by #93.
